frappe/integrations/doctype/ldap_settings/ldap_settings.py

```diff
@@ -13,10 +13,16 @@ def validate(self):
 			return
 
```

Comment:
BUG: if the user enters invalid ldap attributes an exception is thrown and not presented to the user

Come up with a solution

Reply:
implemented attribute validation in 1b2ec4ff775e6d99373be566a872cbe253ba7264